.github/workflows/build.yml

```diff
@@ -107,11 +107,11 @@ jobs:
       - name: install rustup components
         run: rustup component add rustfmt
 
-      - name: install cargo-cache
-        run: cargo install --locked cargo-cache --version 0.8.3
-
-      - name: install cargo-sweep
-        run: cargo install --locked cargo-sweep --version 0.7.0
+      - name: cargo install tools
```

Comment:
Do you want to add these to the stellar/binaries repo and install them prebuilt from there?

Reply:
oh maybe! though I think they get cached

Reply:
How do these installs get cached in the namespace runner? I don't have much knowledge about how caching works here.